src/types.ts

```diff
@@ -21,6 +21,8 @@ export interface MoviePreference {
 export interface Movie {
   externalMovieId: string;
   title: string;
+  overview: string;
```

Comment:
Should `genres` and `status` be added here as well?

Reply:
Not sure why I put those in there...